### PR TITLE
chore: Try to apply customSSLCert via simctl if possible

### PR DESCRIPTION
**Pull request by mykola-mokhnach**

### CODE REVIEW

File: lib/driver.js [hunk 1]
```diff
@@ -371,20 +371,24 @@ class XCUITestDriver extends BaseDriver {
...
-          await this.startSim();
-          this.logEvent('customCertInstalled');
+        try {
+          await this.opts.device.simctl.addRootCertificate(this.opts.customSSLCert);
+        } catch (ign) {
+          if (await hasSSLCert(this.opts.customSSLCert, this.opts.udid)) {
+            log.info(`SSL cert '${_.truncate(this.opts.customSSLCert, {length: 20})}' already installed`);
+          } else {
+            log.info(`Installing ssl cert '${_.truncate(this.opts.customSSLCert, {length: 20})}'`);
+            await shutdownSimulator(this.opts.device);
```
Comment (KazuCocoa) [inline lib/driver.js]: To ensure the device stopped?

Comment (mykola-mokhnach) [reply lib/driver.js]: yes, it was there before

File: lib/driver.js [hunk 2]
```diff
@@ -371,20 +371,24 @@ class XCUITestDriver extends BaseDriver {
...
-          await shutdownSimulator(this.opts.device);
-          await installSSLCert(this.opts.customSSLCert, this.opts.udid);
-          log.info(`Restarting Simulator so that SSL certificate installation takes effect`);
-          await this.startSim();
-          this.logEvent('customCertInstalled');
+        try {
+          await this.opts.device.simctl.addRootCertificate(this.opts.customSSLCert, {raw: true});
+        } catch (ign) {
+          if (await hasSSLCert(this.opts.customSSLCert, this.opts.udid)) {
+            log.info(`SSL cert '${_.truncate(this.opts.customSSLCert, {length: 20})}' already installed`);
```
Comment (jlipps) [inline lib/driver.js]: we're doing this truncating twice, does it make sense to only do it once?

Comment (mykola-mokhnach) [reply lib/driver.js]: changed

**Pull request merged**